rest/src/routes/networkRoutes.js

```diff
@@ -62,12 +68,15 @@ module.exports = {
 
 		server.get('/network/fees/transaction', (req, res, next) => {
 			const numBlocksTransactionFeeStats = services.config.numBlocksTransactionFeeStats || 1;
-			return db.latestBlocksFeeMultiplier(numBlocksTransactionFeeStats).then(feeMultipliers => {
+			const nodeProperties = readAndParseNodePropertiesFile();
+			const latestBlocksFeeMultiplier = db.latestBlocksFeeMultiplier(numBlocksTransactionFeeStats);
+			return Promise.all([nodeProperties, latestBlocksFeeMultiplier]).then(feeMultipliers => {
 				res.send({
-					averageFeeMultiplier: Math.floor(average(feeMultipliers)),
-					medianFeeMultiplier: Math.floor(median(feeMultipliers)),
-					highestFeeMultiplier: Math.max(...feeMultipliers),
-					lowestFeeMultiplier: Math.min(...feeMultipliers)
+					averageFeeMultiplier: Math.floor(average(feeMultipliers[1])),
+					medianFeeMultiplier: Math.floor(median(feeMultipliers[1])),
+					highestFeeMultiplier: Math.max(...feeMultipliers[1]),
+					lowestFeeMultiplier: Math.min(...feeMultipliers[1]),
+					minFeeMultiplier: Number(feeMultipliers[0].node.minFeeMultiplier.replace(`'`, ''))
```

Comment:
nodeMinFeeMultiplier? This value would be different for different nodes.

Reply:
we can leave it like this, and comment in the openAPI spec I guess?

Reply:
Ok, fair, keep minFeeMultiplier